### ISSUE TITLE
Request: CARC and BRRES file support

### ISSUE THREAD

**Issue by Souperman9** [KillzXGaming/Switch-Toolbox#273]
It would great to have support for carc and brres files. In Wii Fit U, most of the files are carcs with brres's inside, as well as kcls that don't seem to load.

**Comment by LolHacksRule** [comment]
I don't think Wii Fit U uses CARCs, it uses JARCs/JTEX, also send the files to get more help

**Comment by Souperman9** [comment]
[Jogging.zip](https://github.com/KillzXGaming/Switch-Toolbox/files/5066257/Jogging.zip)
Here's a CARC that doesn't open - in any program at all. This BRRES that I extracted from a different one seems to be a JARC, judging by the magic number. However, I haven't found anything that can open it either. The same goes for all of the JMDLS

**Comment by LolHacksRule** [comment]
That CARC has absolutely nothing inside, after Yaz0 decomping... The BRRES (jARC) I think is zlib compressed.

**Comment by Souperman9** [comment]
Yeah, I see the CARC is empty... don't know how I didn't notice that before. As far as I can tell though, the JARC doesn't have any compression. It just seems to be an average archive.

**Comment by Souperman9** [comment]
The kcl's successfully load after the kcl rewrite.

**Comment by AtomicLugia** [comment]
I would love to see support for opening Wii Fit U's files (JARC, JMDL, JTEX and JMOT)! I know it's not a Switch game but still.
